parts/k8s/kubernetesagentcustomdata.yml

```diff
@@ -107,6 +107,14 @@ write_files:
     RemainAfterExit=yes
     ExecStart=/usr/local/bin/health-monitor.sh container-runtime
 
+- path: /etc/apt/preferences.d/proposed-updates
```

Comment:
is there a Bionic equivalent for 18.04? Just curious on how I can integrate this with the 18.04 PR